lib/known_bugs.pm

```diff
@@ -65,6 +65,7 @@ sub create_list_of_serial_failures {
         push @$serial_failures, {type => $type, message => 'Kernel stack is corrupted', pattern => quotemeta 'stack-protector: Kernel stack is corrupted'};
         push @$serial_failures, {type => $type, message => 'Kernel BUG found', pattern => quotemeta 'BUG: failure at'};
         push @$serial_failures, {type => $type, message => 'Kernel Ooops found', pattern => quotemeta '-[ cut here ]-'};
+        push @$serial_failures, {type => $type, message => 'Kernel Call Trace found', pattern => quotemeta 'Call Trace:'};
```

Comment:
I've just notice that every test which intentionally triggers OOM killer will produce a call trace in dmesg log. That'll also trigger this check. We should either set `type => soft` here or drop the PR entirely.

Reply:
@mdoucha FYI updated to soft.